webpack.config.js

```diff
@@ -0,0 +1,38 @@
+const path = require('path');
+
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+const webpack = require('webpack');
+
+module.exports = {
+  mode: process.env.NODE_ENV || 'development',
+  entry: './src/index.mjs',
+  output: {
+    path: path.join(__dirname, 'public'),
+    filename: 'wowser-client-[chunkhash:8].js',
+  },
+  resolve: {
+    extensions: ['.js', '.mjs'],
```

Comment:
@fallenoak Thoughts on whether we should use `.mjs` or not?

For Node.js code, it gets rid of a compilation step to support `import/export` by using the `--experimental-modules`-flag: https://nodejs.org/api/esm.html

Browsers and webpack don't really care what the extension is.

Reply:
👍